phoenix-core/src/it/java/org/apache/phoenix/end2end/SpillableGroupByIT.java

```diff
@@ -76,7 +76,7 @@ public static void doSetup() throws Exception {
         
         // Set guidepost width, but disable stats
         props.put(QueryServices.STATS_GUIDEPOST_WIDTH_BYTES_ATTRIB, Long.toString(20));
-        props.put(QueryServices.STATS_ENABLED_ATTRIB, Boolean.toString(false));
+        props.put(QueryServices.STATS_COLLECTION_ENABLED, Boolean.toString(false));
```

Comment:
Is it better to also set STATS_ENABLED_ATTRIB to be false here, although it is deprecated.